app/models/site_config.rb

```diff
@@ -19,7 +19,13 @@ class SiteConfig < RailsSettings::Base
   # staff account
   field :staff_user_id, type: :integer, default: 1
   field :default_site_email, type: :string, default: "yo@dev.to"
-  field :social_networks_handle, type: :string, default: "thepracticaldev"
```

Comment:
Since we are removing the default we need to set the new values in the console post deployment. I don't think this can be done with a data script as its very DEV specific in its values, correct me if I'm wrong though?

Reply:
We can do it with a temporary task yeah! See https://github.com/thepracticaldev/dev.to/pull/7172/files#diff-5572c24b41d78468e9beaf06590e3a31 as an example

Reply:
Done!